docs/docs/queries/query-types.md

```diff
@@ -1,204 +1,486 @@
+
+!!! summary "`TASK` Query Type"
+    `TASK` queries render an interactive list of all tasks in your vault. `TASK` Queries are executed on **task level**, not page level, allowing for task-specific filtering. This is the only command in dataview that modifies your original files if interacted with.
+
+~~~
+```dataview
+TASK
+```
+~~~
+
+**Output**
+
+- [ ] Buy new shoes #shopping
+- [ ] Mail Paul about training schedule 
+- [ ] Finish the math assignment
+    - [x] Finish Paper 1 [due:: 2022-08-13]
+    - [ ] Read again through chapter 3 [due:: 2022-09-01]
+    - [x] Write a cheatsheet [due:: 2022-08-02]
+    - [ ] Write a summary of chapter 1-4 [due:: 2022-09-12]
+- [x] Hand in physics
+- [ ] Get new pillows for mom #shopping
+- [x] Buy some working pencils #shopping
+
+You can use [data commands](data-commands.md) like for all other Query Types. Data Commands are executed on task level, making [implicit fields on tasks](../annotation/metadata-tasks.md) directly available. 
+
+~~~
+```dataview
+TASK
+WHERE !completed AND contains(tags, "#shopping")
+```
+~~~
+
+**Output**
+
+- [ ] Buy new shoes #shopping
+- [ ] Get new pillows for mom #shopping
+
+A common use case for tasks is to **group tasks by their originating file**:
+
+~~~
+```dataview
+TASK
+WHERE !completed
+GROUP BY file.link
+```
+~~~
+
+**Output**
+
+[2022-07-30](#) (1)
+
+- [ ] Finish the math assignment
+    - [ ] Read again through chapter 3 [due:: 2022-09-01]
+    - [ ] Write a summary of chapter 1-4 [due:: 2022-09-12]
+
+[2022-09-21](#) (2)
+
+- [ ] Buy new shoes #shopping
+- [ ] Mail Paul about training schedule
+
+[2022-09-27](#) (1)
+
+- [ ] Get new pillows for mom #shopping
+
+!!! hint "Counting tasks with subtask"
+    Noticing the (1) on the header of `2022-07-30`? Child tasks belong to their parent task and are not counted separately. Also, they do **behave differently** on filtering.
+
+### Child Tasks
+
+A task is considered **child task** if it is **intended by a tab** and below a not indented task list item.
+
+- [ ] clean up the house
+	- [ ] kitchen
+	- [x] living room
+	- [ ] Bedroom [urgent:: true]
+
+
+!!! info "Childs of a bullet point item"
+    While intended tasks under a bullet point item are strictly speaking also children task, dataview will handle them like normal tasks in most cases.
+
+Child Tasks **belong to their parent**. This means if you're querying for tasks, you'll get child tasks as part of their parent back.
+
+~~~
+```dataview
+TASK
+```
+~~~
+
+**Output**
+
+- [ ] clean up the house
+	- [ ] kitchen
+	- [x] living room
+	- [ ] Bedroom [urgent:: true]
+- [ ] Call the insurance about the car
+- [x] Find out the transaction number
+
+This especially means that child task will be part of your result set **as long as the parent matches the query** - even if the child task itself doesn't.
+
+~~~
+```dataview
+TASK
+WHERE !completed
+```
+~~~
+
+**Output**
+
```

Comment:
Should we add a one-liner here saying "Note how `living room` task shows up in the result even though it's complete."?